dvc/exceptions.py

```diff
@@ -179,8 +179,8 @@ def __init__(self, stages):
 class ConfirmRemoveError(DvcException):
     def __init__(self, path):
         super(ConfirmRemoveError, self).__init__(
-            "unable to remove '{}' without a confirmation from the user. Use "
-            "'-f' to force.".format(path)
+            "unable to remove or overwrite '{}' without a confirmation "
```

Comment:
I wouldn't reuse `ConfirmRemoveError` since it is about safely removing files on things like checkout, which doesn't quite fit `config overwrite` usecase :slightly_smiling_face:

Reply:
So, considering the previous comment will return as it was